lib/game.rb

```diff
@@ -0,0 +1,9 @@
+class Game
+
+  attr_reader :name, :cost, :national_drawing
+  def initialize(name, cost, national_drawing = false)
+    @name = name
+    @cost = cost
+    @national_drawing = national_drawing
```

Comment:
this, according to the interaction pattern, needs a question mark method `national_drawing?` and so you can't just use an attr_reader for this one.  make sure to follow the pattern really closely.